templates/app/support.phtml

```diff
@@ -28,20 +28,35 @@
             </p>
 
             <p>
-                <a href="https://funding.communitybridge.org/projects/laminas-project"
+                <a href="https://crowdfunding.lfx.linuxfoundation.org/projects/laminas-project"
                    class="btn btn-secondary">
-                    Become a sponsor via CommunityBridge <i class="fas fa-chevron-right"></i>
+                    Become a sponsor via LXF Crowdfunding <i class="fas fa-chevron-right"></i>
                 </a>
             </p>
         </div>
 
         <div class="col-md-4 col-sm-12 order-1 order-sm-2">
-            <div class="support-banner">
-                <a href="https://funding.communitybridge.org/projects/laminas-project">
-                    <img src="https://funding.communitybridge.org/assets/cb-f-horizontal-white.svg">
+            <div class="bg-white border border-mvc p-4">
+                <a href="https://crowdfunding.lfx.linuxfoundation.org/projects/laminas-project" class="text-mvc">
+                    <img class="img-fluid" src="/images/lxf-crowdfunding.svg">
                     Support Laminas Developers Directly
                 </a>
             </div>
         </div>
     </div>
+    <div class="row align-items-start mt-5">
+        <div class="col-md-8 col-sm-12 order-2 order-sm-1 mt-5 mt-s-0 mt-md-0 mt-lg-0 mt-xl-0">
+            <h2>Commercial Vendor Program</h2>
+            <p>
+                Commercial companies and organisations that wish to support the ongoing development and maintenace of
+                the Laminas projects can apply to join our new commercial vendor program
+            </p>
+        </div>
+
+        <div class="col-md-4 col-sm-12 order-1 order-sm-2">
+            <a class="btn btn-secondary w-100 p-4" href="<?= $this->url('app.commercial-vendor-program') ?>">
+                Find out more
+            </a>
+        </div>
+    </div>
```

Comment:
This creates a new layout but consistency is the goal. Therefore remove the row and the columns and move the content to the first column of this page, under this link:

```html
<p>
    <a href="https://crowdfunding.lfx.linuxfoundation.org/projects/laminas-project"
       class="btn btn-secondary text-white">
        Become a sponsor via LXF Crowdfunding <i class="fas fa-chevron-right"></i>
    </a>
</p>
```

You can use the following:

```html
<h2 class="mt-5">Commercial Vendor Program</h2>
<p>
    Commercial companies and organisations that wish to support the ongoing development and maintenace of
    the Laminas projects can apply to join our new commercial vendor program
</p>

<a class="btn btn-secondary text-white" href="<?= $this->url('app.commercial-vendor-program') ?>">
    Find out more
</a>
```